Update publishing guidelines

walshbr opened the pull request:
Closes #1137.

@mdlincoln already approved the changes in ticket, so I don't think his review is required. 

Asking for a review from @alsalin since she will be the first person to actually follow the managing editor changes. Take a look and see if they make sense? I'll gather everything up for the FFMPEG lesson on the ph-submissions repo in just a couple minutes, and then you can take a swing at the publishing process whenever you're ready. Feel free to ping me by email or on slack if you wanted to talk while doing it. 

Once @alsalin approves the text here, the changes will need to be translated in Spanish (and potentially French, depending on where you're at in the process?), though the changes to the text itself are minimal. It's mostly moving things around.

walshbr committed:
Account for shift in ph_authors.yml changes to ME

walshbr committed:
add slug example and restore slug description

walshbr committed:
Oop messed up the numbering

walshbr committed:
Update editorial guideliens

walshbr committed:
Update to reiterate twitter botting

alsalin reviewed (dismissed):
It looks good in theory! Let's hope I can actually follow it. I may try to take a crack at the first lesson today.

arojascastro commented:
All these changes concern new tasks for managing editors, so I guess the most suitable person in the Spanish team for translating this is @mariajoafana  - let us know if you need help.

spapastamkou commented:
We can integrate the changes once we make a final review of the corresponding FR page before publishing the web site, you can go on with the rest @walshbr (if @mdlincoln sees no problem us doing this way)

walshbr committed:
Oop. We use dashes not hyphens.

walshbr commented:
Oop I just noticed the tests were failing. Fixed, and they should be good for translating and then merging once Spanish is done!

alsalin commented:
Questions/comments - now that I've gone through this twice (using the web GH), I realize the directions for dealing with file moving, copying, deleting are not clear. Mostly using "copy," we ask the ME to create new files in jekyll/, but should we add a line to delete the old files once all this moving, publishing, and archiving is done?

walshbr commented:
@alsalin sorry just saw this. Which files were you talking about deleting? Was there something not captured below? 

```
Go to the directory for your local ph-submissions repository.
Add a new line to the YAML header of the now published lesson: redirect_from: "/lessons/LESSON-SLUG"
Copy the now published lesson from lessons/ into lessons/published/.
Copy the image folder containing the images for the now published lesson from images/ to images/published/.
Use git add, git commit, and git push to finalize all the changes (or follow the Making Technical Contributions instructions: https://github.com/programminghistorian/jekyll/wiki/Making-Technical-Contributions)

```
Also just a note to say that once we've wrapped the English portion of this I'll be stepping back for the sabbatical. Seems like @mariajoafana and the French team have plans for who will be translating this. Sounds like once Spanish translates this can be merged, and the French version will be incorporated once they go live?

alsalin commented:
@walshbr the directions are for the command line and refer to "copying" rather than "moving" - in that sense, you end up with multiple copies in different folders

walshbr commented:
Aaah I see what you're saying @alsalin. That does seem like a good catch. Would just changing the word "copy" to "move" do it?

alsalin commented:
@walshbr i think so! as someone who is terrified of deleting things always, being clear on that language would help :P

walshbr committed:
Change copy to move

walshbr commented:
These changes should be good for English and can be merged once Spanish has them translated. I'll step back and sabbatical now. :)

acrymble commented:
@mariajoafana you've been volunteered to do this translation. Are you able to do that or do you need someone else to take it on?

mdlincoln committed:
Merge branch 'gh-pages' into update-publishing-guidelines

mdlincoln commented:
I've just updated this rather old branch against the latest site, so it now has all the new French pages in it. I think this means that both @programminghistorian/spanish-team and @programminghistorian/french-team should now update their translations of the publishing guidelines to this branch?

spapastamkou committed:
Update consignes-redacteurs.md

updates publishing guidelines in FR, fixes typos and one internal link, adds minor changes in translation

mariajoafana commented on en/editor-guidelines.md:
````diff
@@ -197,7 +197,7 @@ Once you and the author are happy with a tutorial, the next step is to begin the
+If the lesson has been written by a new author, the managing editor will need to add a new bio for that person. You will need to provide the managing editor with a version of the following information:
 
 ```yaml
 - name: Jim Clifford
````
this section includes information on the lesson but no space for adding the actual bio

mariajoafana commented:
I'm working on the translation into Spanish.

mariajoafana committed:
Update guia-editor.md (#1192)

* update-affiliation-sofia #1169 (#1171)

* update-affiliation-sofia #1169

* fix little typo

* Issue 901 - Spanish worldcat entry (#1178)

* add spanish worldcat entry to /contribute

* add /es worldcat entry.

* Update contribuciones.md

* typo fix

* add spanish worldcat entry.

* format tweak

* fix typos

* fix spelling in FR page

* Update guia-editor.md

* use hyphen in ph-submissions, not underscore

* fix typos

* fix url

* fix typo
